packages/vm/src/runTx.ts

```diff
@@ -326,15 +326,6 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
       throw new Error(msg)
     }
     const parentBlock = await this.blockchain.getBlock(opts.block?.header.parentHash)
-    if (parentBlock === null) {
-      const msg = _errorMsg(
-        `Parent block not found in blockchain so cannot compute data gas price`,
-        this,
-        block,
-        tx
-      )
-      throw new Error(msg)
-    }
```

Comment:
I was a bit lazy here (since code could not be reached before anyhow), we could also expand here though and wrap in a try/catch and recreate the error message if wished.